app/legacy/space-migration/space-migrator.ts

```diff
@@ -24,19 +29,22 @@ export default class SpaceMigrator {
 
   async migrate(onUpdate) {
     await fs.ensureDir(this.destDir)
+    const cmd = await pathsClient.brimcap()
     return new Promise<void>((resolve, reject) => {
-      this.process = spawn("zdeps/brimcap", [
+      this.process = spawn(cmd, [
         "migrate",
         `-zqd=${this.srcDir}`,
         `-root=${this.destDir}`
       ])
+
       const linesErr = readline.createInterface({input: this.process.stderr})
       let total = 0
       let count = 0
       let space = ""
       let totalRegexp = /^migrating (\d+) spaces$/
 
       linesErr.on("line", (line) => {
+        console.log(line)
```

Comment:
Was this just for debug or did you want to keep this in?

Reply:
I want that in, in case the user needs more info about a problem.